src/cli/serve/integration_tests/invalid_config.test.ts

```diff
@@ -41,7 +41,7 @@ describe('cli invalid config support', () => {
           .split('\n')
           .filter(Boolean)
           .map((line) => JSON.parse(line) as LogEntry)
-          .filter((line) => line.log.level === 'FATAL');
+          .filter((line) => line.log?.level === 'FATAL');
```

Comment:
The ECS logs from the agent now have a `"log.level": level` entry instead of `"log": { "level": level }`, messing with this test